app/Post_Types/Archive_Item/AdminModifications.php

```diff
@@ -13,6 +17,31 @@ public function hooks() {
 		add_filter( 'admin_body_class', array( &$this,'admin_body_class') );
 		add_filter( 'manage_edit-diviner_archive_item_columns', [ $this, 'archival_item_columns' ] );
 		add_action( 'manage_diviner_archive_item_posts_custom_column', [ $this, 'manage_diviner_archive_item_posts_custom_column' ], 10, 2 );
+		add_action( 'carbon_fields_register_fields', [ $this, 'active_field_setup' ], 3, 0 );
+	}
+
+	function active_field_setup(  ) {
+		// get active fields
+		$field_query = new \WP_Query( array(
```

Comment:
Here I'd use a `get_posts` call rather than a `WP_Query` one. Under the hood `get_posts` will do the same thing but will set `suppress_filters` to `false` by default.
Being this a query for a business-logic CPT reducing the filters (they are not *really* suppressed, but reduced) seems legit.
The code following this one should be modified to accomodate for that.

Reply:
And the `get_posts` call should really fetch just `ids` using the `fields` parameter, the arguments you can keep as they are.
```php
$meta_query = array( 'active-only' => array('key' => Helper::get_real_field_name(FieldPostMeta::FIELD_ACTIVE ), 'value' => FieldPostMeta::FIELD_CHECKBOX_VALUE ) );
$args = array('fields' => 'ids', 'post_type' => Diviner_Field::NAME, 'meta_query' => $meta_query);
$field_ids = get_posts($args);
```

Reply:
Very thankful for this review @lucatume Good stuff